.circleci/config.yml

```diff
@@ -141,8 +151,23 @@ jobs:
               # Internal publishing from an external CircleCI is... not a thing.
               # curl -s --fail $DOCS_URL | bash -s -- -r docs/requirements.txt $CIRCLE_BRANCH
             fi
+      - run:
+          command: |
+            # hack - notion of "owners" isn't supported in Circle 2
+            if [ $CIRCLE_PROJECT_USERNAME = 'palantir' ] && [ -z $CIRCLE_PR_NUMBER ] && [ -z $CIRCLE_TAG ]; then
+              ./scripts/circle-ci/publish-images-to-antithesis.sh
```

Comment:
I still think long term we should publish a version before tagging images to Antithesis repo. But willing to consider otherwise.

Main concern of doing otherwise would be blocking our ability to easily tag releases on stability of third party software. If for whatever reason things are down on their end, don't want us to have to change the order of these statements to actually be able to get releases published.

Reply:
Makes sense. 

Long term spitballing, but it would be nice to have some way to recall a version then if antithesis fails and we can just click a button to approve the recall :D